### PR TITLE
RDISCROWD-4839: Refactor app_settings, settings_local

### PR DESCRIPTION
**Pull request by dchhabda**
*Issue number of the reported bug or feature request: #<number>*

**Describe your changes**
A clear and concise description of the changes you have made.

**Testing performed**
Describe the testing you have performed to ensure that the bug has been addressed, or that the new feature works as planned.

**Additional context**
Add any other context about your contribution here.

### CODE REVIEW

File: .github/workflows/github-actions.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ jobs:
           redis-server --version
           redis-server contrib/redis/sentinel.conf --sentinel
           cd pybossa && ln -s themes/default/translations && cd ..
-          cp settings_test.py.tmpl settings_test.py
+          cp settings_test.py.tmpl pybossa/settings_test.py
```
Comment (ywrsusan) [inline .github/workflows/github-actions.yml:43]: I think the path should be `/pybossa/settings_test.py`, but with this change the path will be `/pybossa/pybossa/settings_test.py`, is this expected?

Comment (ywrsusan) [reply .github/workflows/github-actions.yml:43]: nvm I saw you changed `run_tests`

**Review by ywrsusan — APPROVED** [review]
Think that you will do a test deploy before merging, right? Looks good for the test deploy.

File: .github/workflows/github-actions.yml (hunk 1 above)
Comment (dchhabda) [reply .github/workflows/github-actions.yml:43]: right. also, this was not required but `nice to have` all settings at once place.

File: pybossa/view/diagnostics.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
 from flask import Blueprint, Response, current_app
 
 from pybossa.core import sentinel, db, talisman
-from pybossa.hdfs.client import HDFSKerberos
+from pybossa.pybhdfs.client import HDFSKerberos
```
Comment (XiChenn) [inline pybossa/view/diagnostics.py:24]: looks to me the previous name "hdfs" is more concise.

**Review by ywrsusan — APPROVED** [review]

**Pull request merged**